src/pages/AddNotes.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from "react";
+import "../css/note.css";
+
+const AddNotes = (props) => {
+  const [expand, setExpand] = useState(false);
+  const [note, setNote] = useState({
+    title: "",
+    content: "",
+  });
+
+  const InputEvent = (event) => {
+    const { value, name } = event.target;
+
+    setNote((prevData) => {
+      return {
+        ...prevData,
+        [name]: value,
+      };
+    });
+
+    console.log(note);
+  };
+
+  const addNoteHandler = () => {
+    props.passNote(note);
+    setNote({
+      title: "",
+      content: "",
+    });
+  };
```

Comment:
you can de-structure props at the top so that you don't need to write props everywhere like, ```{passNote} ```
Also you can make initial state for the note  and then re-use it while re-setting instead of passing ``` {
      title: "",
      content: "",
    }``` multiple times